main.go

```diff
@@ -80,13 +80,20 @@ func main() {
 	localServer1 := &httputil.ReverseProxy{Director: localConf1.ProxyDirector}
 	localServer2 := &httputil.ReverseProxy{Director: localConf2.ProxyDirector}
 
+	errorChan := make(chan error, 1)
+
 	// setup node 1 proxy to forward request to node 2 in case of error response
 	proxyServer1 := &httputil.ReverseProxy{
 		Director:       conf1.ProxyDirector,
 		ModifyResponse: conf1.ModifyResponse,
 		ErrorHandler: func(writer http.ResponseWriter, r *http.Request, err error) {
 			logger.Error("node1 failed to respond", zap.Error(err))
 
+			// known http2 error, cannot recover once this error is hit. only solution is to restart pod
+			if strings.Contains(err.Error(), "after Request.Body was written; define Request.GetBody to avoid this error") {
+				errorChan <- err
+			}
```

Comment:
Can we add a `FIXME` here + a link to the relevant GitHub issue? Pretty hacky so wouldn't want this logic to exist forever

Reply:
Yea sure!